src/memote/Memote.jl

```diff
@@ -0,0 +1,289 @@
+"""
+# Memote
+
+Main entry point into metabolic model quality tests.
+"""
+module Memote
+
+using DocStringExtensions
+using COBREXA
+using Test
+using JuMP
+using ..Utils
+
+include(joinpath("Config.jl"))
+include(joinpath("checks", "Annotation.jl"))
+include(joinpath("checks", "Basic.jl"))
+include(joinpath("checks", "Biomass.jl"))
+include(joinpath("checks", "Consistency.jl"))
+include(joinpath("checks", "Energy.jl"))
+include(joinpath("checks", "GPRAssociation.jl"))
+include(joinpath("checks", "Metabolite.jl"))
+include(joinpath("checks", "Network.jl"))
+include(joinpath("checks", "Reaction.jl"))
+
+using .Config
+using .Annotation
+using .Basic
+using .Biomass
+using .Consistency
+using .Energy
+using .GPRAssociation
+using .Metabolite
+using .Network
+using .Reaction
```

Comment:
if you don't have exports you don't need `using` -- the prefix below will make sure this passes ok.